Update serving tutorial (#4815)
knative/docs: docs/snippets/quickstart-install.md

    1. Install Knative and Kubernetes in a minikube instance by running:

        !!! note
            The minikube cluster will be created with 6&nbsp;GB of RAM. If you don't have enough memory, you can change to a
            different value not lower than 3&nbsp;GB by running the command `minikube config set memory 3078` before this command.
        ```bash
        kn quickstart minikube
        ```

    1. The output of the previous command asked you to run minikube tunnel.
       Run the following command to start the process in a secondary terminal window, then return to the primary window and press enter to continue:
        ```bash
        minikube tunnel --profile knative
        ```

>         The tunnel must continue to run in a terminal window while you are using your Knative `quickstart` environment at any time.

Reviewer: ```suggestion
        The tunnel must continue to run in a terminal window any time you are using your Knative `quickstart` environment.
```

>         The tunnel command is required as it would allow Knative ingress service to be accesible as a LoadBalancer from your host computer.

Reviewer: ```suggestion
        The tunnel command is required because it allows your cluster to access Knative ingress service as a LoadBalancer from your host computer.
```

        ```bash
        kn quickstart minikube
        ```
        !!! note
            The minikube cluster will be created with 6GB of RAM, if you don't have enough memory you can change to a

>             different value not lower than 3GB with the command `minikube config set memory 3078` before this command.

Author: yes they need to run it before `kn quickstart minikube`

    1. During the previous command you will be asked to run the minikube tunnel.

>        Start the process in a secondary terminal window and and return to the primary window and press enter to continue :

Author: thats a good idea
